pkg/scheduler/framework/interface_test.go

```diff
@@ -291,15 +291,15 @@ func TestIsStatusEqual(t *testing.T) {
 			want: false,
 		},
 		{
-			name: "wrap error status should be equal with original one",
+			name: "wrap error status should not be equal with original one",
```

Comment:
wait, this should be true, if we change `errorStatus = AsStatus(errors.New("internal error"))`